blog/2024-11-03-options-flow.md

```diff
@@ -0,0 +1,12 @@
+---
+author: epenet
+authorURL: https://github.com/epenet
+title: "New properties for options flows, and deprecated OptionsFlowWithConfigEntry"
+---
+
+New helper properties have been added to the `OptionsFlow` to facilitate access to the config entry:
+- `self._config_entry_id` gives you access to the ID of the config entry
+- `self.config_entry` gives you access to the config entry
+- `self.options` gives you access to a mutable copy of the config entry
```

Comment:
a mutable copy of the config entry options?

Reply:
Yes. As it's a copy you can change it.